modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -14,12 +14,12 @@
 
 /obj/machinery/self_actualization_device
 	name = "Self-Actualization Device"
-	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Morningstar corporation \
-	has teamed up with the Cinco Chemical & Toy Division to bring you the Self-Actualization Device! \
+	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Veymed(ical) Corporation \
```

Comment:
Why the `(ical)` here? That looks a bit odd.

Reply:
Yea, in lore, the company is known as VeyMed, and is only VeyMedical Corporation in official documents, according to current lore document on the company's parent empire (non-public for now iirc).